edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/DeviceScopeIdentitiesCacheTest.cs

```diff
@@ -240,8 +240,8 @@ public async Task RefreshCacheWithRefreshRequestTest()
             deviceScopeIdentitiesCache.InitiateCacheRefresh();
             deviceScopeIdentitiesCache.InitiateCacheRefresh();
 
-            // Wait for the 2 refresh cycles to complete, this time because of the refresh request
-            await Task.Delay(TimeSpan.FromSeconds(2));
+            // Wait for the 5 refresh cycles to complete, this time because of the refresh request
```

Comment:
This should be 2, not 5

Reply:
It is not the the time, but the number of times we try..

Reply:
reverted